manager/integration/tests/test_settings.py

```diff
@@ -782,3 +786,169 @@ def test_setting_concurrent_rebuild_limit():  # NOQA
     (Not sure if we need to put this specific case as a separate test in
      test_engine_upgrade.py)
    """
+
+    # Step 1-1
+    update_setting(client,
+                   "concurrent-replica-rebuild-per-node-limit",
+                   "1")
+
+    # Step 1-2
+    volume1_name = "test-vol-1"  # NOQA
+    volume1 = create_and_check_volume(client, volume1_name, size=str(3 * Gi))
+    volume1.attach(hostId=get_self_host_id())
+    volume1 = wait_for_volume_healthy(client, volume1_name)
+
+    volume2_name = "test-vol-2"  # NOQA
+    volume2 = create_and_check_volume(client, volume2_name, size=str(3 * Gi))
+    volume2.attach(hostId=get_self_host_id())
+    volume2 = wait_for_volume_healthy(client, volume2_name)
+
+    # Step 1-3
+    os.system("dd if=/dev/urandom of=/dev/longhorn/{} bs=1M count=2000".
+              format(volume1_name))
+    os.system("dd if=/dev/urandom of=/dev/longhorn/{} bs=1M count=2000".
+              format(volume2_name))
+
+    # Step 1-4, 1-5
+    delete_replica_on_test_node(client, volume1_name)
+    wait_for_rebuild_start(client, volume1_name)
+    delete_replica_on_test_node(client, volume2_name)
+    wait_for_rebuild_complete(client, volume1_name)
+    wait_for_rebuild_start(client, volume2_name)
+    wait_for_rebuild_complete(client, volume2_name)
```

Comment:
The steps here doesn't seem to follow the test skeleton steps 4-5. According to the test skeleton we should:
```
delete_replica_on_test_node(client, volume1_name)
delete_replica_on_test_node(client, volume2_name)
wait_for_rebuild_start(client, volume1_name)
while volume1 is rebuilding:
   verify that volume 2 is not rebuilding 
wait_for_rebuild_start(client, volume2_name)
```

Reply:
HI @PhanLe1010 
I added loop to make sure when volume1 is rebuilding, volume 2 is not rebuilding